src/exts/tvdb_info/main.py

```diff
@@ -0,0 +1,126 @@
+from typing import Literal
+
+import aiohttp
+import discord
+from discord import ApplicationContext, Bot, Cog, option, slash_command
+
+from src.settings import THETVDB_COPYRIGHT_FOOTER, THETVDB_LOGO
+from src.tvdb import TvdbClient
+from src.tvdb.models import SearchResults
+from src.utils.log import get_logger
+
+log = get_logger(__name__)
+
+MOVIE_EMOJI = "🎬"
+SERIES_EMOJI = "📺"
+
+
+class InfoView(discord.ui.View):
+    """View for displaying information about a movie or series."""
+
+    def __init__(self, results: SearchResults):
+        super().__init__(disable_on_timeout=True)
+        self.results: SearchResults = results
+        self.dropdown = discord.ui.Select(
+            placeholder="Not what you're looking for? Select a different result.",
+            options=[
+                discord.SelectOption(
+                    label=result.name or result.title or "",
+                    value=str(i),
+                    description=result.overview[:100] if result.overview else None,
+                )
+                for i, result in enumerate(self.results)
+            ],
+        )
+        self.dropdown.callback = self._dropdown_callback
+        self.add_item(self.dropdown)
+        self.index = 0
+
+    def _get_embed(self) -> discord.Embed:
+        result = self.results[self.index]
+        original_title = result.name
+        en_title = result.translations["eng"]
+        original_overview = result.overview
+        en_overview = result.overviews["eng"]
+        if en_overview and en_overview != original_overview:
+            overview = f"{en_overview}"
+        elif not en_overview and original_overview:
+            overview = f"{original_overview}\n\n*No English overview available.*"
+        else:
+            overview = "*No overview available.*"
+        if original_title and original_title != en_title:
+            title = f"{original_title} ({en_title})"
+        else:
+            title = en_title
+        if result.id[0] == "m":
+            title = f"{MOVIE_EMOJI} {title}"
+            url = f"https://www.thetvdb.com/movies/{result.slug}"
+        else:
+            title = f"{SERIES_EMOJI} {title}"
+            url = f"https://www.thetvdb.com/series/{result.slug}"
+        embed = discord.Embed(title=title, color=discord.Color.blurple(), url=url)
+        embed.add_field(name="Overview", value=overview, inline=False)
+        embed.set_footer(text=THETVDB_COPYRIGHT_FOOTER, icon_url=THETVDB_LOGO)
+        embed.set_image(url=result.image_url)
+        return embed
+
+    async def _dropdown_callback(self, interaction: discord.Interaction) -> None:
+        if not self.dropdown.values or not isinstance(self.dropdown.values[0], str):  # noqa: PD011
```

Comment:
Huh, is this really a pandas object or is ruff just drunk?

Reply:
It's just drunk i have no f-ing idea what was going on there

Reply:
Okay, let's just globally ignore the `PD` ruff rules in pyproject, we don't need them for this app anyways, super weird, I'll try and get a minimal repro from this to report to ruff

Reply:
@ItsDrike Can you make that and then I'll rebase on main or do I add it from this pr?

Reply:
c0923e66cd7410d167b04b9c6d6aa59d699a4edc